docs/drupal_twig_debugging.md

```diff
@@ -0,0 +1,36 @@
+# Drupal Twig Debugging
+
+[Drupal.org: Debugging Twig templates](https://www.drupal.org/docs/theming-drupal/twig-in-drupal/debugging-twig-templates)
+
+## Locating Template files
+
+[Drupal.org: Locating Template Files with Debugging](https://www.drupal.org/node/2358785)
+
+### For Mass.gov
+
+1. Uncomment the Twig debugging lines in the file `docroot/sites/default/services.local.yml`.
+2. Clear the cache by running `ddev drush cr`.
+
+**Note: Don't commit changes to the `services.local.yml` file.**
+
+## Viewing variables
+
+[Drupal.org: Viewing variables](https://www.drupal.org/docs/theming-drupal/twig-in-drupal/debugging-twig-templates#s-viewing-variables)
+
+### For Mass.gov using Xdebug
+
+Initial setup:
+1. [Setup Xdebug](https://github.com/massgov/openmass#xdebug) for your IDE.
```

Comment:
this link points to a broken link and the instruction doesn't work https://github.com/massgov/openmass/blob/.env.example

Reply:
```suggestion
1. Enable xdebug in ddev by running `ddev xdebug on`.  See [PHP Step Debugging](https://ddev.readthedocs.io/en/stable/users/step-debugging/) for ddev specific setup instructions and how to configure your IDE
```